lib/process-services-cloud/src/lib/process/start-process/components/start-process-cloud.component.ts

```diff
@@ -373,7 +374,11 @@ export class StartProcessCloudComponent implements OnChanges, OnInit, OnDestroy
     }
 
     processDefinitionSelectionChanged(processDefinition) {
-        this.processDefinitionSelection.emit(processDefinition);
+        if (processDefinition) {
+            const defaultProcessName = this.processNamePipe.transform(this.name, processDefinition.name);
```

Comment:
I'd like to have something like 
```
processNamePipe.transform('Example - %{processDefinition} - %{datetime}', {processDefinition: 'upload passport'});
```

Where the second param is the ProcessDetailsModel